apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/LoadedBatch.java

```diff
@@ -153,16 +153,14 @@ public static LoadedBatch combine(LoadedBatch a, LoadedBatch b) {
    * slightly simpler and, since conversion is done once, just as efficient.
    */
   private static String convertToString(List<String> list) {
-    if (list == null || list.isEmpty()) {
-      return "";
-    }
-    return list.stream().collect(Collectors.joining(SEPARATOR));
+    return (list == null || list.isEmpty())
+        ? ""
+        : list.stream().collect(Collectors.joining(SEPARATOR));
   }
 
   private static List<String> convertToList(String commaSeparated) {
-    if (commaSeparated == null || commaSeparated.isEmpty()) {
-      return new ArrayList<>();
-    }
-    return Arrays.asList(commaSeparated.split(SEPARATOR, -1));
+    return (commaSeparated == null || commaSeparated.isEmpty())
+        ? new ArrayList<>()
+        : Arrays.asList(commaSeparated.split(SEPARATOR, -1));
```

Comment:
I don't see a need to make this change

Reply:
pet peeve on my part; I have seeing a lot of return statements if not needed.